pkg/daemon/metrics.go

```diff
@@ -23,7 +23,7 @@ var (
 	// MCDSSHAccessed shows ssh access count for a node
 	MCDSSHAccessed = prometheus.NewCounter(
 		prometheus.CounterOpts{
-			Name: "ssh_accessed",
+			Name: "ssh_accessed_total",
```

Comment:
```suggestion
			Name: "ssh_successful_logins_total",
```

Reply:
When a user directly perform ssh, we annotate node with `machineconfiguration.openshift.io/ssh:accessed` , this was done in the beginning of OCP 4 to discourage users from directly doing ssh into nodes and making any changes. I think, keeping metric name ssh_accessed_total would be less confusing as it aligns with why its usage.

Reply:
Will update help message accordingly.

Reply:
usually a counter metric should be named like `<noun>_total`, not `<adjective>_total` (e.g. `http_requests_total` , not `http_requested_total`). Maybe `ssh_accesses_total`?

Reply:
updated metric name to `ssh_accesses_total`